### ISSUE TITLE
navigate with tab in recipe ingredients

### ISSUE THREAD

**Issue by TWART016** [TandoorRecipes/recipes#1305]
**Is your feature request related to a problem? Please describe.**
I create a recipe by myself or want to change an existing one. In the Ingredients area open the `Unit` dropdown and search for `ml` with the keyboard. With up/down I can select the right one and then have to click enter to select. With Tab I can jump to the next field `Food`

**Describe the solution you'd like**
When selecting the Unit with the Keyboard and `Tab` the selected unit will be written in the field an jump to the next field `Food`.

**Comment by vabene1111** [comment]
i am not sure if this is possible with the library we are using, but you could just use enter to make a selection and tab afterwards or is that not working for you ?

**Comment by TWART016** [comment]
Yes, that's working. But these are 2 keys instead of one and it can improve the usability if it's possible with the library.

**Comment by Szeraax** [comment]
I think the idea is that if you are highlighting an ingredient, that could be considered as already selected, hence tab should just keep the currently highlighted (and therefore, selected) ingredient. This is problematic if selecting an ingredient does any database calls vs if just highlighting does.
